vtex-apps/google-tag-manager
# feat/adding avaiable flag to the product datalayer

#### What is the purpose of this pull request?

 Adding a flag to the Product Data Layer to identify if the product is available or not. 

#### What problem is this solving?

Today the Product Data Layer Object does not differentiate between available and unavailable products. 
With this flag we are allowing customers to Filter this on their Google analytics.
The main use case is when working with pageViews on Google Analytics, customers today don't have a visibility of what products are available and not. offering this capability will allow them to focalice efforts to the products that are not available but receiving tons of visits. 

#### How should this be manually tested?
This Product is unavailable: https://tagmanager--arcaplanet.myvtex.com/collare-rogz-nylon-snake-mis-m-rossa-mm-16/p
1.- right click and inspect 
2.- go to console and type window.dataLayer
3.- open the productDetail event and check the flag isAvailable=false

This product is available: https://tagmanager--arcaplanet.myvtex.com/seresto-collare-antiparassitario-cani-oltre-8-kg/p
1.- right click and inspect 
2.- go to console and type window.dataLayer
3.- open the productDetail event and check the flag isAvailable=true
